packages/wmr/src/plugins/npm-plugin/resolve.js

```diff
@@ -1,3 +1,21 @@
+import { resolve as _resolveExports, legacy as _resolveLegacyEntry } from 'resolve.exports';
+
+function resolveExports(pkg, key) {
+	const conditions = [process.env.NODE_ENV === 'production' ? 'production' : 'development', 'esmodules', 'module'];
+
+	return _resolveExports(pkg, key, {
+		browser: true,
+		conditions
+	});
+}
+
+function resolveLegacyEntry(pkg, path) {
+	_resolveLegacyEntry(pkg, {
+		browser: path,
+		fields: ['esmodules', 'modern', 'module', 'jsnext:main']
```

Comment:
```suggestion
		fields: ['esmodules', 'modern', 'module', 'jsnext:main', 'main']
```

Reply:
^ still incomplete actually, this will push `browser` to the end of the priority order, which doesnt match the original priority list